flask_monitoringdashboard/__init__.py

```diff
@@ -76,6 +78,11 @@ def bind(app, schedule=True):
 
     atexit.register(flush_cache)
 
+    if micro:
```

Comment:
why do we need a teardown only on micro? 
shouldn't this be in there always?

Reply:
So this is here to help with https://github.com/flask-dashboard/Flask-MonitoringDashboard/issues/276. 
The reason it was happening for me was because I am using one dashboard for multiple apps and the cache flush only gets triggered on the overview call of the api. This meant the last call time was always out of date unless I went to that apps dashboard. 
I have it under micro because that is when the problem is most prevelent and assumed someone would want to test it properly in a non micro context.